app/assets/javascripts/search.js

```diff
@@ -5,18 +5,21 @@ if($("#advanced-search").length){
   var $description = $('input#description');
   var $downloads   = $('input#downloads');
   var $updated     = $('input#updated');
+  var $yanked      = $('input#yanked');
 
   $name.add($summary)
     .add($description)
     .add($downloads)
     .add($updated)
+    .add($yanked)
     .on('input', function(e) {
-      var name        = $name.val().length > 0 ? 'name: ' + $name.val() : '';
-      var summary     = $summary.val().length > 0 ? 'summary: ' + $summary.val() : '';
-      var description = $description.val().length > 0 ? 'description: ' + $description.val() : '';
-      var downloads   = $downloads.val().length > 0 ? 'downloads: ' + $downloads.val() : '';
-      var updated     = $updated.val().length > 0 ? 'updated: ' + $updated.val() : '';
+      var name        = $name.val().length > 0 ? 'name: ' + $name.val() + ' ' : '';
+      var summary     = $summary.val().length > 0 ? 'summary: ' + $summary.val() + ' ' : '';
+      var description = $description.val().length > 0 ? 'description: ' + $description.val() + ' ' : '';
+      var downloads   = $downloads.val().length > 0 ? 'downloads: ' + $downloads.val() + ' ' : '';
+      var updated     = $updated.val().length > 0 ? 'updated: ' + $updated.val() + ' ' : '';
+      var yanked      = $yanked.is(":checked") ? 'yanked: true' : '';
 
-      $main.val($.trim(name + ' ' + summary + ' ' + description + ' ' + downloads + ' ' + updated));
+      $main.val($.trim(name + summary + description + downloads + updated + yanked));
```

Comment:
why are we removing space again? if it is not related to yanked search result, can you please send a separate PR for it?

Reply:
This just avoids extra spaces coming up in the query string.
For eg. if summary is not used n the filter, unnecessary space if added.
This doesn't affect the search results though. I just did this for JS cleanup that we discussed.

Reply:
Should I make a separate PR?

Reply:
okay, didn't notice you moved space to var. You can leave it here but ideally, separate PR for unrelated changes is the best.